server/middleware/__tests__/caseType.spec.js

```diff
@@ -1,19 +1,17 @@
 import { createHeaders } from '../../models/user';
 
 jest.mock('../../clients/index');
-jest.mock('../../libs/logger');
 jest.mock('../../models/user');
 const { infoService } = require('../../clients/index');
 const { getCaseType, getCaseTypes } = require('../caseType');
-const getLogger = require('../../libs/logger');
 
 const headers = '__headers__';
 const req = { params: { type: '__type__' }, user: '__user__' };
 const sendStatus = jest.fn();
 const json = jest.fn();
 const res = { json, locals: {}, sendStatus };
 const next = jest.fn();
-const caseTypes = [{ label: 'Case Type 1', value: 'caseType1' }, { label: 'Case Type 2', value: 'caseType2' }];
+const caseTypes = [ { label: 'Case Type 1', value: 'caseType1' }, { label: 'Case Type 2', value: 'caseType2' } ];
```

Comment:
You've put otherwise unrelated cosmetic changes into the same commit as everything else. Consider splitting them into two commits (you can use `git add -p` for this).

Reply:
https://github.com/UKHomeOffice/hocs-management-ui/pull/327#issuecomment-1301886429